cypress/integration/models/developer/applicationinventory/application.ts

```diff
@@ -220,8 +220,18 @@ export class Application {
     }
 
     delete(cancel = false): void {
+        selectItemsPerPage(100);
         cy.wait(2000);
-        performRowAction(this.name, deleteAction);
+        cy.get(tdTag)
```

Comment:
performRowAction method is doing the exact same thing, then why are we changing this ?

Reply:
@sshveta , the performRowAction() function was updated recently and it broke the application delete functionality . I've fixed the delete method now . The PR is ready for review.